modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-pricing-impl/rest-openapi-v2.0.yaml

```diff
@@ -762,7 +766,7 @@ components:
                     items:
                         $ref: "#/components/schemas/TierPrice"
                     type: array
-                unitOfMeasure:
+                unitOfMeasureKey:
```

Comment:
aligned this name change with https://github.com/liferay-commerce/liferay-portal/pull/4185